### DISCUSSION TITLE
fortune最近好像出错了该怎么改啊

### DISCUSSION THREAD

**Discussion by fankongqunxing** [nonebot/discussions#357] [问答]
fortune最近好像出错了该怎么改啊

**Comment by yanyongyu** [comment]
什么出错，出了什么错

**Reply by fankongqunxing** [reply]
![2023-03-27 22 23 13](https://user-images.githubusercontent.com/124325245/227969406-eba2c1a3-e1e4-467b-854a-55f8a4fac195.png)

**Reply by fankongqunxing** [reply]
一装上fortune就这样 群友说可以改 我不太会

**Reply by yanyongyu** [reply]
fortune插件的问题你不应该在这问
